app/assets/stylesheets/_tables.scss

```diff
@@ -0,0 +1,36 @@
+[aria-sort] > * {
+  position: relative;
+
+  &:after {
+    font-size: em(10, 19);
+    position: absolute;
+    right: -15px;
+    top: 50%;
+    margin-top: -5px;
+  }
+
+  &:visited {
+    color: $link-colour;
+  }
+
+  &:hover {
+    color: $link-hover-colour;
+  }
+}
+
+[aria-sort=descending],
+[aria-sort=ascending] {
+    padding-right: em(30, 19);
+}
+
+[aria-sort=descending] > * {
+  &:after {
+    content: '\00a0\00a0\25B2';
```

Comment:
Removed unnecessary space characters as elements is positioned by it's right hand side so these are useless.